src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -594,78 +594,93 @@ const schema = {
     },
     form0781: {
       type: 'object',
-      incident: {
-        type: 'array',
-        items: {
-          type: 'object',
-          properties: {
-            personalAssault: {
-              type: 'boolean'
-            },
-            medalsCitations: {
-              type: 'string'
-            },
-            incidentDate: {
-              $ref: '#/definitions/date'
-            },
-            incidentLocation: {
-              type: 'string'
-            },
-            incidentDescription: {
-              type: 'string'
-            },
-            unitAssigned: {
-              type: 'string'
-            },
-            unitAssignedDates: {
-              $ref: '#/definitions/dateRange'
-            },
-            remarks: {
-              type: 'string'
-            },
-            additionalChanges: {
-              type: 'string'
-            },
-            personInvolved: {
-              type: 'array',
-              items: {
-                type: 'object',
-                name: {
-                  $ref: '#/definitions/fullName'
-                },
-                rank: {
-                  type: 'string'
-                },
-                injuryDeath: {
-                  type: 'string',
-                  enum: [
-                    'Killed in Action',
-                    'Killed Non-Battle',
-                    'Wounded in Action',
-                    'Injured Non-Battle',
-                    'Other'
-                  ]
-                },
-                injuryDeathOther: {
-                  type: 'string'
-                },
-                injuryDeathDate: {
-                  $ref: '#/definitions/date'
-                },
-                unitAssigned: {
-                  type: 'string'
+      properties: {
+        "remarks": {
+          "type": "string"
+        },
+        "additionalIncidentText": {
+          "type": "string"
+        },
+        "additionalSecondaryIncidentText": {
+          "type": "string"
+        },
+        "otherInformation": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        incident: {
+          type: 'array',
+          minItems: 1,
```

Comment:
Do we need minItems? You can have `form0781: null` as a valid payload

Reply:
I know we talked about this yesterday, and I think you were right.  The minItems will only be enforced when we have form781 element, which will not happen unless we have an incident.